src/Api/Auth/Models/Request/WebAuthn/WebAuthnLoginCredentialCreatelRequestModel.cs

```diff
@@ -29,5 +29,8 @@ public class WebAuthnCredentialRequestModel
     [EncryptedString]
     [EncryptedStringLength(2000)]
     public string EncryptedPrivateKey { get; set; }
+
+    public (string userKey, string publicKey, string privateKey) GetKeys()
+        => (EncryptedUserKey, EncryptedPublicKey, EncryptedPrivateKey);
```

Comment:
question: I like, but it's not used? WIP maybe? 😄

Reply:
I think it was vestigial from a previous implementation I was working on. 🤔